Android/app/src/main/java/entity/orderList/Dish.java

```diff
@@ -256,4 +256,14 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(getName(), getPrice(), getCategory(), getIngredients(), getFeatures(), getCalories(), getAllergyInformation(), getStatus(), getTableNum());
     }
+
+    public void increasePrice() {
```

Comment:
the code is fine, but would it be better for the manager to be able to change the price to a specific amount? E.g. change price to exactly $4.50 from original $3.00? Nothing wrong with the current implementation, just curious